homeassistant/components/garmin_connect/__init__.py

```diff
@@ -73,38 +75,33 @@ def __init__(self, hass, client):
         self.client = client
         self.data = None
 
-    async def _get_combined_alarms_of_all_devices(self):
-        """Combine the list of active alarms from all garmin devices."""
-        alarms = []
-        devices = await self.hass.async_add_executor_job(self.client.get_devices)
-        for device in devices:
-            device_settings = await self.hass.async_add_executor_job(
-                self.client.get_device_settings, device["deviceId"]
-            )
-            alarms += device_settings["alarms"]
-        return alarms
-
-    @Throttle(MIN_SCAN_INTERVAL)
+    @Throttle(DEFAULT_UPDATE_INTERVAL)
     async def async_update(self):
-        """Update data via library."""
+        """Update data via API wrapper."""
         today = date.today()
+        summary = None
```

Comment:
We don't need to define these placeholders.

Reply:
Good one, removed them, thanks.